docs/contribution/community-components.md

```diff
@@ -0,0 +1,299 @@
+
+### Component checklist
+
+This check-list consists of a series of agreed upon acceptance criteria (put
+forth by Carbon designers and developers) that lives in the `package.json` file
+as a set of boolean config objects. Each spec will need some meta data (i.e.
+first major release when it was introduced) before we can badge it.
+Contributions can then be evaluated according to the completion of the spec.
+Some of the criteria can be tested through automation, some will be
+self-reported.
+
+In order to contribute a community component, the only _truly_ required items
+are the basic guidelines. This allows us to give you a unique package directory
+in the monorepo. Reaching 100% completeness means meeting all of the design and
+development guidelines on this list.
+
+**Basic (always required)**
+
+- [ ] Unique design or feature
+- [ ] Unique component name
+
+**Design artifacts (added in [community box folder](link here))**
+
+- [ ] Component drawing (in Sketch / XD / Sigma) template link
+- [ ] States: default, hover, active, disabled, etc
+- [ ] Component in-situ product screens
+- [ ] Component in-situ flows detailing the job-to-be-done
+- [ ] Responsive design on all breakpoints
+- [ ] Component redline / spec (template link)
+- [ ] Component Usage guidance (writing)
+- [ ] Design explorations, audit, and research
+
+**Visual**
+
+- [ ] Using IBM type styles
+- [ ] Using IBM colors
+- [ ] Using IBM icons
+- [ ] Using IBM grid & layout guidance
+- [ ] Design in all 4 color themes
+- [ ] Satisfy basic accessibility contrast requirements (text readability 4.5:1,
+      info graphics 3:1)
+
+**Code**
+
+- [ ] Built with Carbon v10+
+- [ ] SASS
+- [ ] Using Carbon design tokens (color, type, motion, spacing)
+- [ ] `#{$prefix}` class naming
+- [ ] Semantic HTML
+- [ ] Code documentation (props, class/selectors)
+- [ ] Include different states (hover, focus, disabled)
+- [ ] Unit testing
+- [ ] Should have motion (CSS transition)
+- [ ] Theming functionality
+
+**Responsiveness**
+
+- [ ] Mobile (`sm`, `md`)
+- [ ] Desktop (`lg`, `xlg`)
+- [ ] Large format (`max`)
+
+**Accessibility**
+
+- [ ] Pass AVT 1 test
+- [ ] Pass AVT 2 test
+- [ ] Pass AVT 3 test
+
+**Frameworks**
+
+- [ ] Vanilla JS
+- [ ] React
+- [ ] Angular
+- [ ] Vue
+
+**Browser support**
+
+- [ ] Chrome
+- [ ] Safari
+- [ ] Firefox
+- [ ] IE11
+- [ ] Edge
+
+### Process
+
+1. **Contributor License Agreement:** Before you can contribute any code, we
+   need you to sign a Contributor License Agreement (CLA).
+2. **Development environment:** If you haven't already, fork and clone the
+   community components monorepo. Checkout our
+   [Developer Handbook](../developer-handbook.md) to read up on our best coding
+   practices and proper commit messages.
+3. **Issue:** Check repo for an _existing_ issue related to your contribution
+   first. If none exist, open a new issue.
+4. **Package directory:** Once you open an issue, we will review that you have a
```

Comment:
Out of curiosity, what would the turn around time be on creating the set up for contributions.

Besides a duplicate name would there be any reason why an issue would be denied?